app/assets/javascripts/discourse/app/pre-initializers/inject-discourse-objects.js

```diff
@@ -50,19 +51,29 @@ export default {
     app.register("current-user:main", currentUser, { instantiate: false });
     app.currentUser = currentUser;
 
-    ALL_TARGETS.forEach((t) =>
-      app.inject(t, "topicTrackingState", "topic-tracking-state:main")
-    );
+    ALL_TARGETS.forEach((t) => {
+      app.inject(t, "topicTrackingState", "topic-tracking-state:main");
+      app.inject(t, "pmTopicTrackingState", "pm-topic-tracking-state:main");
+    });
 
     const topicTrackingState = TopicTrackingState.create({
       messageBus: MessageBus,
       siteSettings,
       currentUser,
     });
+
     app.register("topic-tracking-state:main", topicTrackingState, {
       instantiate: false,
     });
 
+    const pmTopicTrackingState = PrivateMessageTopicTrackingState.create({
```

Comment:
You could inject `currentUser` here like the other tracking state, then you won't need to do `User.current()` when calling `startTracking`

Reply:
Good point :+1: